src/c++/perf_analyzer/inference_profiler.h

```diff
@@ -319,6 +324,21 @@ class InferenceProfiler {
   cb::Error ProfileHelper(
       const bool clean_starts, PerfStatus& status_summary, bool* is_stable);
 
+  /// A helper function to determine stability
```

Comment:
Just from a cursory glance, this function says it determines stability, but there's another function below (`check_stability()`) that also seems like it determines stability based on its name.

Can one or both of these functions have their names/documentation updated to more clearly highlight the differences in what they're doing?

Reply:
done